scripts/monitoring/cron-send-logging-checks.py

```diff
@@ -0,0 +1,224 @@
+        node_cmd = "exec -ti {} -- {}".format(es_master_name, curl_cmd)
+        cluster_nodes = self.oc.run_user_cmd(node_cmd)['nodes']
+        es_status['all_nodes_registered'] = 1
+        # The internal ES node name is a random string we do not track anywhere
+        # pylint: disable=unused-variable
+        for node, data in cluster_nodes.items():
+        # pylint: enable=unused-variable
+            has_matched = False
+            for pod in es_pods:
+                if data['host'] == pod['metadata']['name']:
+                    has_matched = True
+
+            if has_matched == False:
+                es_status['all_nodes_registered'] = 0
+
+        return es_status
+
+    def check_fluentd(self):
+        ''' Verify fluentd is running '''
+        fluentd_status = {}
+
+        # Get all nodes with fluentd label
+        nodes = self.oc.get_nodes()
+        fluentd_nodes = []
+        for node in nodes['items']:
+            if 'logging-infra-fluentd' in node['metadata']['labels']:
+                if node['metadata']['labels']['logging-infra-fluentd'] == 'true':
+                    fluentd_nodes.append(node)
+
+        # Get all fluentd pods
+        pods = self.oc.get_pods()
+        fluentd_pods = []
+        for pod in pods['items']:
+            if 'component' in pod['metadata']['labels']:
+                if pod['metadata']['labels']['component'] == 'fluentd':
+                    fluentd_pods.append(pod)
+
+        # Make sure fluentd is on all the nodes and the pods are running
+        fluentd_status['number_expected_pods'] = len(fluentd_nodes)
+        fluentd_status['number_pods'] = len(fluentd_pods)
+        fluentd_status['node_mismatch'] = 0
+        for node in fluentd_nodes:
+            node_matched = False
+            for pod in fluentd_pods:
+                if node['metadata']['labels']['kubernetes.io/hostname'] == pod['spec']['host']:
+                    node_matched = True
+
+            if node_matched == False:
+                fluentd_status['node_mismatch'] = 1
+
+        fluentd_status['running'] = 1
+        for pod in fluentd_pods:
+            if pod['status']['containerStatuses'][0]['ready'] == False:
+                fluentd_status['running'] = 0
+
+        return fluentd_status
+
+    def check_kibana(self):
+        ''' Check to see if kibana is up and working '''
+        kibana_status = {}
+
+        # Get logging url
+        route = self.oc.get_route('logging-kibana')['status']['ingress'][0]['host']
+        kibana_url = "https://{}/".format(route)
+
+        # Disable SSL to work around self signed clusters
+        ctx = ssl.create_default_context()
+        ctx.check_hostname = False
+        ctx.verify_mode = ssl.CERT_NONE
+
+        # Verify that the url is returning a valid response
+        kibana_status['site_up'] = 1
+        try:
+            res = urllib2.urlopen(kibana_url, context=ctx)
+            print res
```

Comment:
Could consider using `try/except/else` here and placing the `print res` under the else clause.